config/hardware/ercf.cfg

```diff
@@ -4,13 +4,6 @@
 variable_ercf_enabled: True
 gcode:
 
-# Include all files which are created by the install script of Happy Hare
-[include ../../ercf_hardware.cfg]
-[include ../../ercf_menu.cfg]
-[include ../../ercf_software.cfg]
-[include ../../ercf_parameters.cfg]
-[include ../../ercf_vars.cfg] ## TODO: check if this is better placed centrally in klippain and not introduced only via ERCF in klippain
```

Comment:
If you remove that from here. How do you include the HH files?

Reply:
I include them in printer.cfg. because of the symlink it's not possible to make otherwise :(
unless you have a genius idea.

Reply:
We could put an ercf.cfg in the mainfolder so we have only one include in printer.cfg

Reply:
It's another solution

Reply:
I know it's not very clean but I've never found a solution to this problem with the symlink

Reply:
how do we handle klipperscreen happy hare edition? :D if we are already talking about optionals :P

Reply:
I don't need to add anything to klippain for install HH KlipperScreen (same for crowsnest...)

Reply:
Oh well, symlinks... Ok this change a little bit the thing yes... I'll try to think about how to do it properly :)